test/chainweb/pact/send_request_body_test.exs

```diff
@@ -6,14 +6,16 @@ defmodule Kadena.Chainweb.Pact.SendRequestBodyTest do
   use ExUnit.Case
 
   alias Kadena.Chainweb.Pact.SendRequestBody
-  alias Kadena.Types.Command
+  alias Kadena.Types.{Command, Signature}
 
   setup do
     command =
       Command.new(
         hash: "-1npoTU2Mi71pKE_oteJiJuHuXTXxoObJm8zzVRK2pk",
         sigs: [
-          "8b234b83570359e52188cceb301036a2a7b255171e856fd550cac687a946f18fbfc0e769fd8393dda44d6d04c31b531eaf35efb3b78b5e40fd857a743133030d"
+          Signature.new(
```

Comment:
I was wondering why in this part you use the `.new("SIG")` and in the other tests you used the `%Signature{sign: "SIG"` notation?

Reply:
I just tried to follow the style that each test had.